picard/ui/options/plugins.py

```diff
@@ -97,6 +107,10 @@ def add_plugin_item(self, plugin, enabled=False, item=None):
             item.setCheckState(0, QtCore.Qt.Checked)
         else:
             item.setCheckState(0, QtCore.Qt.Unchecked)
+        if bold:
```

Comment:
Imho we should have 2 colums for versions, one listing the current version, one the newest available version, and a download button for each updateable plugin. Get rid of ambiguous checkboxes.

Somethink like:
 plugin title | authors | Current version | Latest version | Enabled 
 ------------- | --------- | ----------- | ------------- | ----------
 title1 | author1 | 1.1.0 | 1.2.0 [Update] | [x] 
 title2 | author2 | 0.1.0 | 0.1.0 | [x] 
 title3 | author3 |  0.2.0 | 0.2.0 | [  ] 

Where plugin is enabled/disabled by a checkbox (if not installed, it will be installed on first check)
Button to update existing plugin appears when needed only